e2e/process-services-cloud/edit-process-filters-component.e2e.ts

```diff
@@ -95,6 +95,8 @@ describe('Edit process filters cloud', () => {
             processCloudDemoPage.editProcessFilterCloudComponent().clickSaveAsButton();
             processCloudDemoPage.editProcessFilterCloudComponent().editProcessFilterDialog().setFilterName('New').clickOnSaveButton();
 
+            browser.driver.sleep(1000);
```

Comment:
Isn't there any alternative than using all those browser.driver.sleep ?

Reply:
probably yes i will create a separate task to remove it later